DataConnectors/CrowdStrike Falcon Endpoint Protection/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "CrowdStrikeFalconEndpointProtection",
+    "title": "CrowdStrike Falcon Endpoint Protection",
+    "publisher": "CrowdStrike",
+    "descriptionMarkdown": "The [CrowdStrike Falcon Endpoint Protection](https://www.crowdstrike.com/endpoint-security-products/) connector allows you to easily connect your CrowdStrike Falcon Event Stream with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization's endpoints and improves your security operation capabilities.",
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **CrowdStrikeFalconEventStream** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-crowdstrikefalcon-parser)",
```

Comment:
Please use https://aka.ms/sentinel-crowdstrikefalconendpointprotection-parser for parser redirection link.

Reply:
resolved